static/js/invite.js

```diff
@@ -118,7 +118,16 @@ function generate_multiuse_invite() {
         success(data) {
             const copy_link_html = copy_invite_link(data);
             ui_report.success(copy_link_html, invite_status);
-            new ClipboardJS("#copy_generated_invite_link");
+            const clipboard = new ClipboardJS("#copy_generated_invite_link");
+            clipboard.on("success", () => {
+                feedback_widget.show({
+                    populate(container) {
+                        container.html(i18n.t("Invite-link copied to clipboard."));
+                    },
+                    title_text: i18n.t("Invite-link"),
+                });
+                $(".feedback_undo").css("display", "None");
```

Comment:
You can do just `$(".feedback_undo").hide()`. But, what I really think we should do is add a prep commit which modifies feedback_widget.show, if there is no `undo` function passed, we hide `feedback_undo` by default.

Reply:
I was thinking of modifying the feedback.show function but then opted for the simpler option. Thanks for the review, I will try to execute the same.